spec.bs

```diff
@@ -116,6 +116,15 @@ Take the handwritten text "int" for example:
   If the application wants to delete character "t", it can remove stroke 3 and 4 from the drawing.
 </div>
 
+<h2 id="api-idioms">API Idioms</h2>
+
+The [=task source=] mentioned in this specification is the <dfn data-export id="handwriting-recognition-task-source">handwriting recognition task source</dfn>.
+
+When an algorithm <dfn data-export id="queue-task">queues a Handwriting Recognition API task</dfn> |T|, the user agent *MUST* [=queue a global task=] |T| on the [=handwriting recognition task source=] using the [=global object=] of the [[ecma-262#current-realm|current realm]].
+
+Unless specified, the [[ecma-262#realm|realm]] for JavaScript objects constructed by algorithm steps is the [[ecma-262#current-realm|current realm]].
```

Comment:
Same as above to avoid more brittle fragment references:

```suggestion
Unless specified, the [=ECMAScript/realm=] for JavaScript objects constructed by algorithm steps is [=the current realm record=].
```